synapse/handlers/room_list.py

```diff
@@ -381,9 +381,13 @@ async def get_remote_public_room_list(
                     Codes.UNRECOGNIZED,
                     Codes.NOT_FOUND,
                 ):
+                    # Not an error that should trigger a fallback.
                     logger.debug("Falling back to locally-filtered /publicRooms")
```

Comment:
Is this right? The comment says the error shouldn't trigger a fallback but the following log line says it's triggering a fallback.

Reply:
I put this comment in the wrong place...